features/git-sync/current_branch/feature_branch/folder_does_not_exist_on_main_branch/no_conflict.feature

```diff
@@ -0,0 +1,65 @@
+Feature: git sync: syncing inside a folder that doesn't exist on the main branch (with open changes)
+
+  As a developer syncing inside a committed folder that doesn't exist on the main branch
+  I want the command to at least finish properly
+  So that my repo is left in a consistent state and I don't lose any data
+
+
+  Background:
+    Given I have feature branches named "current_feature" and "other_feature"
+    And the following commits exist in my repository
+      | BRANCH          | LOCATION         | MESSAGE              | FILE NAME        |
+      | main            | local and remote | main commit          | main_file        |
+      | current_feature | local and remote | folder commit        | new_folder/file1 |
+      | other_feature   | local and remote | other feature commit | file2            |
+    And I am on the "current_feature" branch
+    And I have an uncommitted file
+    When I run `git sync --all` in the "new_folder" folder
```

Comment:
Why is this `git sync --all`. it should be `git sync` (based on the location of the feature), which means the `other_feature` branch can be removed and `current_feature` branch can be renamed to `feature`